src/interpreter.rs

```diff
@@ -358,7 +363,11 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
                                                 arg_srcs.push((src, ty));
                                             }
                                         }
-                                        ty => panic!("expected tuple as last argument in function with 'rust-call' ABI, got {:?}", ty),
+                                        ty => {
+                                            panic!("expected tuple as last argument in function \
+                                                    with 'rust-call' ABI, got {:?}",
+                                                   ty)
+                                        }
```

Comment:
So, this one is interesting. I prefer to break the line-length limit for error message strings so that they are greppable. Newlines mess this up if I tried to grep `function with 'rust-call'`.

On the other hand, I've been meaning to extract this entire block of code out into a separate function for a while now, which will remove indentation and help with things like this and the above diff I commented on.